shared/src/markup.rs

```diff
@@ -4,12 +4,14 @@ use std::collections::HashSet;
 pub enum NodeType {
     Text(String),
     Binding(String),
+    RepeatBinding(RepeatBindingData),
     Group,
     Button(ButtonData),
     LineInput(LineInputData),
     ProgressBar(ProgressBarData),
     Template(TemplateData),
-    Repeat(RepeatData),
+    UnlinkedRepeat(UnlinkedRepeatData),
```

Comment:
Those changes on Repeat should be reverted to what they were before. There should be just one enum value as before and not three.